### ISSUE TITLE
[T9][Accessibility][Functional]:Unable to make selection in familly tab -Behavioral - Expand and collapse

### ISSUE THREAD

**Issue by SR027213** [SR027213/Feb-19-Bug-Bash#32]
Steps to Reproduce:
Unable to make selection in familly tab -Behavioral - Expand and collapse
